dwellingplace/templates/index.html

```diff
@@ -3,12 +3,14 @@
 {% block title %} | Home{% endblock %}
 
 {% block body %}
+
 	<h1>Dwelling Place Visualizer Data Exchange</h1>
+ 
     <h2>Upload Excel Data</h2>
     <section>
         <form action="{{ url_for('index.upload') }}" method="POST" enctype="multipart/form-data">
-            <input type="file" name="file">
-            <button type="submit">Upload</button>
+            <input type="file" name="file" accept="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet">
```

Comment:
On firefox/mac this limits to .xlsx only and does not permit .xls  I think we want to allow .xls files too